pinot-core/src/main/java/org/apache/pinot/core/segment/processing/genericrow/GenericRowFileWriter.java

```diff
@@ -29,7 +29,6 @@
 import org.apache.pinot.spi.data.FieldSpec;
 import org.apache.pinot.spi.data.readers.GenericRow;
 
-
```

Comment:
(minor, code format) When I auto reformat the file, Pinot Style will leave 2 empty lines here